<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DESMainWindow</class>
 <widget class="QMainWindow" name="DESMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DES测试程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>正确性验证</string>
     </attribute>
     <widget class="QLabel" name="clearTextLabel">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>明文(Hex)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="clearText">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="secretKey">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="secretKeyLabel">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密钥(Hex)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="encryptButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>加密</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decryptButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>解密</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ciphertText">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ciphertTextLabel">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密文(Hex)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>选择计算方式</string>
      </property>
      <widget class="QRadioButton" name="fpgaRadioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>FPGA</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="armRadioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ARM</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>信息</string>
      </property>
      <widget class="QLabel" name="timeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>计算时间:</string>
       </property>
      </widget>
      <widget class="QLabel" name="methodLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>实际计算方式:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>高级性能测试</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>181</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>选择计算方式</string>
      </property>
      <widget class="QRadioButton" name="fpgaRadioButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>FPGA</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="armRadioButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ARM</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>141</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>信息</string>
      </property>
      <widget class="QLabel" name="cipherSize">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>密文大小：</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeLabel_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>计算时间:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PlainSize_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>明文大小:</string>
       </property>
      </widget>
      <widget class="QLabel" name="methodLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>实际计算方式:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>110</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>选择文件...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>110</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>加密</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>解密</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>加解密后路径:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
